var/spack/repos/builtin/packages/madis/package.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
I wonder if this addition should be its own PR. That will make it easier to cherry-pick and send to the authoritative repo.

Reply:
I guess it's ok. Thanks for making the changes I requested. Please update from jcsda_emc_spack_stack and we're good to go!